### ISSUE TITLE
Database query error 1064 for second installation?

### ISSUE THREAD

**Issue by q2apro** [q2a/question2answer#595]
I am about to install a second installation with shared userdatabase, but that should not be an issue.

When hitting to the new site, it says to create tables, I hit the button (backend tables categories, sharedevents, userevents, userfavorites, userlevels, usernotices are created), then it brakes with: 

`Database query error 1064
You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'position (position)) ENGINE=InnoDB CHARSET=utf8' at line 1

CREATE TABLE secondsite_pages (pageid SMALLINT UNSIGNED NOT NULL AUTO_INCREMENT, title VARCHAR(80) NOT NULL, nav CHAR(1) CHARACTER SET ascii NOT NULL, position SMALLINT UNSIGNED NOT NULL, flags TINYINT UNSIGNED NOT NULL, permit TINYINT UNSIGNED, tags VARCHAR(200) NOT NULL, heading VARCHAR(800), content MEDIUMTEXT, PRIMARY KEY (pageid), KEY tags (tags), UNIQUE position (position)) ENGINE=InnoDB CHARSET=utf8`

I cannot spot the error in the mysql code. Also it should be a bug in the code then?! 

According to the test here http://rextester.com/WODME10974 the code error should be in the beginning.

**Comment by q2apro** [comment]
According to this answer: https://stackoverflow.com/a/18697128/1066234 there should be no comma before `UNIQUE`.

**Comment by q2apro** [comment]
Seems to be a mysql problem: 

The query above works on localhost with `10.1.19-MariaDB`, but it does not work online with `5.7.21-0ubuntu0.16.04.1-log`.

Version check done using `SELECT @@version;`

**Comment by svivian** [comment]
I'm not able to reproduce this. I'm using the exact same MySQL version

Can you tell me what you get from a `select @@sql_mode` query?

**Comment by pupi1985** [comment]
`position` is not a reserved word. The thing is that when the SQL mode is set to [`IGNORE_SPACE`](https://dev.mysql.com/doc/refman/5.7/en/sql-mode.html#sqlmode_ignore_space) then it is treated as if it was due to the existance of the [`POSITION`](https://dev.mysql.com/doc/refman/5.7/en/string-functions.html#function_position) function.

In order to keep things more compatible across different configurations, it should be backticked (in fact, all of the fields should).

**Comment by q2apro** [comment]
Yes, see stackoverflow link: https://stackoverflow.com/q/48462847/1066234

I backticked the culprits, see commit above.
